### ISSUE TITLE
Gazebo simulation

### ISSUE THREAD

**Issue by Ymonse** [intel-aero/meta-intel-aero#382]
Hello,

Can we use the gazebo simulation without the intel aero ready to fly UAV ? 

I tried to clone that package : git clone git@github.intel.com:lecampos/aero_simulator.git, but I get "ssh: could not resolve hostname github.intel.com".

I'm on a ubuntu 16.04 virtual machine.

Thank you.

**Comment by lucasdemarchi** [comment]
@guermonprez wrong link to the repository? You may need to migrate it under github.com/intel-aero and update the docs.

**Comment by gbarsih** [comment]
I'm having the same problem, could we have a temporary link to the simulator while you update the location?

**Comment by Ymonse** [comment]
Yes it would be great if you could provide a temporary link, because I still have the same problem.

**Comment by stribuda** [comment]
So what is the solution please?
